roles/contiv_network/templates/netplugin.j2

```diff
@@ -0,0 +1,7 @@
+{# If an interface is specified for netplugin, use it #}
+{# Otherwise, hope for the best                       #}
```

Comment:
hmmm, unfortunately I don't agree. This code breaks certain things, how is fixing it incrementally right thing to do?